nodeup/pkg/model/networking/calico.go

```diff
@@ -38,6 +40,23 @@ func (b *CalicoBuilder) Build(c *fi.ModelBuilderContext) error {
 		return nil
 	}
 
+	if networking.Calico.WireguardEnabled {
+		var packages []string
+		if b.Distribution.IsDebianFamily() {
+			packages = append(packages, "wireguard")
+		} else if b.Distribution.IsRHELFamily() {
+			packages = append(packages, "elrepo-release")
+			packages = append(packages, "epel-release")
+			packages = append(packages, "kmod-wireguard")
+			packages = append(packages, "wireguard-tools")
+		} else {
+			klog.Warningf("unknown distribution, skipping wireguard install: %v", b.Distribution)
+		}
+		for _, p := range packages {
+			c.AddTask(&nodetasks.Package{Name: p})
+		}
+	}
```

Comment:
WireGuard should be installed by default when Calico is used to allow switching it on/off without a rolling update of the cluster.
Debian doesn't yet have packages for WireGuard, so we can skip it. RHEL/CentOS require further testing so we can skip it too for now. This leaves us only with Ubuntu.
```suggestion
	if b.Distribution.IsUbuntu() {
		c.AddTask(&nodetasks.Package{Name: "wireguard"})
	}
```